app/controllers/problem_reports_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class ProblemReportsController < ApplicationController
+  def index
+    @problem_reports = ProblemReport.order(id: :desc)
+    @filter = filter_message
+    if @filter == 'Unfixed and Critical'
+      @problem_reports = @problem_reports.where(is_critical: true,
+                                                is_fixed: false)
+    elsif @filter == 'Unfixed and Fixable'
+      @problem_reports = @problem_reports.where(is_fixable: true,
+                                                is_fixed: false)
+    end
+  end
+
+  def new; end
+
+  def create
+    if create_new_report.save
+      redirect_to problem_reports_path, notice: 'Success'
+    else
+      redirect_to new_problem_report_path, notice: 'Fail'
+    end
+  end
+
+  def update
+    update_remarks
+    %w[is_fixable is_fixed is_blocked is_critical].each do |a|
```

Comment:
Don't think you need to do it this way actually. `@report.update(h)`, pass in a hash to `h`